.devcontainer/create-db-user.sql

```diff
@@ -0,0 +1,2 @@
+CREATE USER vscode CREATEDB;
```

Comment:
Is this needed? You use `postgres` user in `.devcontainer/docker-compose.yml`.

Reply:
I was still puzzling over this one myself (it's part of the default boilerplate from this devcontainer generator).  I don't see anything it gets used for, so it's unclear to me why it gets generated.  Since the extra file is annoying, I plan to look into whether it's necessary and strip it out if not. (TODO)